AdvancedChangedFiles/AccountProfile.jsx

```diff
@@ -0,0 +1,346 @@
+                certifications: [],
+                visaStatus: null,
+                visaExpiryDate: null,
+                profilePhoto: null,
+                linkedAccounts: {
+                    linkedIn: "",
+                    github: ""
+                },
+                jobSeekingStatus: null,
+                id: "",
+                firstName: "",
+                middleName: null,
+                lastName: "",
+                gender: null,
+                email: null,
+                phone: null,
+                mobilePhone: null,
+                isMobilePhoneVerified: false,
+                profilePhotoUrl: "",
+                videoName: null,
+                videoUrl: "",
+                cvName: null,
+                cvUrl: "",
+                summary: null,
+                description: null
+
+            },
+            loaderData: loaderData,
+
+        }
+
+        this.updateWithoutSave = this.updateWithoutSave.bind(this)
+        this.updateAndSaveData = this.updateAndSaveData.bind(this)
+        this.updateForComponentId = this.updateForComponentId.bind(this)
+        this.saveProfile = this.saveProfile.bind(this)
+        this.loadData = this.loadData.bind(this)
+        this.init = this.init.bind(this);
+    };
+
+    init() {      
+        var loaderData = Object.assign({}, this.state.loaderData);
+        loaderData.allowedUsers = loaderData.allowedUsers.concat("Talent");
+        loaderData.isLoading = false;
+        this.setState({ loaderData: loaderData }, function () {
+        }.bind(this));
+    }
+
+    componentDidMount() {
+        this.init();
+        this.loadData();
+    }
+
+    loadData() {
+        var cookies = Cookies.get('talentAuthToken');
+        $.ajax({
+            url: 'https://advancedtalentprofile-e2gkffcccyg0hhdm.australiaeast-01.azurewebsites.net/profile/profile/getTalentProfile',
+            headers: {
+                'Authorization': 'Bearer ' + cookies,
+                'Content-Type': 'application/json'
+            },
+            type: "GET",
+            success: function (res) {           
+                if (res && res.data) {
+                    this.updateWithoutSave(res.data);
+                } else {
+                    console.warn('No data returned from the API.');
+                }
+            }.bind(this),
+            error: function (xhr, status, error) {
+                console.error('Error fetching profile data:', error);
+            }
+        });
+    }
+
+    //updates component's state without saving data
+    updateWithoutSave(newValues) {
+
+        let newProfile = Object.assign({}, this.state.profileData, newValues);
+        this.setState({ profileData: newProfile }, () => {
+
+            // Validate the update
+            if (this.state.profileData.id === newProfile.id) {
+                console.log('Profile updated successfully.');
+            } else {
+                console.warn('Profile update failed.');
+            }
```

Comment:
Do you wan to inform the users about this or not? If you want, can you add proper message UI and if not, can you remove these or comment these because we don't want to expose any logs to end-users.